fahrschein/src/main/java/org/zalando/fahrschein/NakadiClient.java

```diff
@@ -85,62 +86,32 @@ public static NakadiClientBuilder builder(URI baseUri) {
     }
 
     /**
-     * Create Subscription for one event type.
+     * Create a subscription for a single event type.
      *
-     * @param applicationName
-     * @param eventName
-     * @param consumerGroup
-     * @return
-     * @throws IOException
+     * @deprecated Use the {@link SubscriptionBuilder} and {@link NakadiClient#subscribe(String, String)} instead.
      */
+    @Deprecated
     public Subscription subscribe(String applicationName, String eventName, String consumerGroup) throws IOException {
-        return subscribe(applicationName,  Collections.singleton(eventName), consumerGroup, SubscriptionRequest.Position.END, Collections.emptyList());
+        return subscribe(applicationName, eventName).withConsumerGroup(consumerGroup).create();
     }
 
     /**
-     * Create Subscription for one event type with specified "read_from" position. If SubscriptionRequest.Position == "cursors"
-     * then initialCursors parameter is required.
-     *
-     * @param applicationName
-     * @param eventName
-     * @param consumerGroup
-     * @param readFrom
-     * @return
-     * @throws IOException
+     * Build a subscription for a single event type.
      */
-    public Subscription subscribe(String applicationName, String eventName, String consumerGroup, SubscriptionRequest.Position readFrom, List<Cursor> initialCursors) throws IOException {
-        return subscribe(applicationName,  Collections.singleton(eventName), consumerGroup, readFrom, initialCursors);
+    public SubscriptionBuilder subscribe(String applicationName, String eventName) throws IOException {
```

Comment:
Or name those `subscription` and the terminal operation `subscribe`, i.e. clients will have `.subscription(applicationName, eventName).withConsumerGroup(group).subscribe()`